ragna/deploy/_api/schemas.py

```diff
@@ -66,6 +66,13 @@ def from_core(cls, message: ragna.core.Message) -> Message:
             sources=[Source.from_core(source) for source in message.sources],
         )
 
+    def to_core(self) -> ragna.core.Message:
+        return ragna.core.Message(
+            content=self.content,
+            role=self.role,
+            sources=[],
+        )
```

Comment:
@pmeier Tell me if this makes sense:
For the purpose of chat history, we don't need Sources for each individual message since they already exist on the chat object. So we don't need to rebuild them here (?)

Reply:
> since they already exist on the chat object

They don't. `Source`'s are returned by the `SourceStorage` and are never saved on the `Chat` but only ever on the `Message`

https://github.com/Quansight/ragna/blob/8ea62a3de0a7131b439b522c8a90587b5b2080ad/ragna/core/_rag.py#L221-L227

If you have a look at the ORM model for the chat, we also have no connection to the sources table, but only the messages

https://github.com/Quansight/ragna/blob/8ea62a3de0a7131b439b522c8a90587b5b2080ad/ragna/deploy/_api/orm.py#L75-L92

The messages table however is connected to the sources table

https://github.com/Quansight/ragna/blob/8ea62a3de0a7131b439b522c8a90587b5b2080ad/ragna/deploy/_api/orm.py#L95-L100